helm/hpcc/templates/_warnings.tpl

```diff
@@ -0,0 +1,59 @@
+{{/*
+
+---  DO NOT EDIT THIS FILE ---
+
+##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+##############################################################################
+
+*/}}
+{{/*
+Generate a list of warnings
```

Comment:
trivial: generally helper function have a line "Pass in ..." documenting what the function expects as input, e.g.:
```
{{/*
A template to generate a PVC
Pass in dict with root, me, name, and optional path
*/}}
```